chat-plugins/chat-monitor.js

```diff
@@ -324,14 +324,20 @@ let commands = {
 				return this.sendReply(`'${words.join(', ')}' ${Chat.plural(words, "were", "was")} removed from the ${list} list.`);
 			}
 		},
+		'': 'view',
```

Comment:
Should maybe show help.

Or maybe the view page should show help?

Reply:
I think it makes sense to show the list here. ``/filters`` to me is: "what are the filters?" I think that's what they'd be for most people that do just that.